<!DOCTYPE html>

<html>
<head>
  <title>Marco Colombo - Experimental area</title>
  <meta charset="utf-8" />
  <meta name="author" content="Marco Colombo" />
  <meta name="description" content="Full steam research in Edinburgh" />
  <link rel="stylesheet" href="/css/style.css" />
  
  <link rel="icon" type="image/png" href="/pics/favicon.png" />
</head>

<body>

<div class="site">

  <div class="header">
    <h1 class="title"><a href="/">Marco Colombo</a></h1>
    <p class="tag">Experimental area &middot; proceed at your own risk!</p>
  </div>

  <div id="navigation">
    <ul>
    
    
      <li><a  class="active" 
           href="/">Home</a></li>
    
      <li><a 
           href="/research/">Research</a></li>
    
      <li><a 
           href="/teaching/">Teaching</a></li>
    
      <li><a 
           href="/software/">Software</a></li>
    
      <li><a 
           href="/engage/">Dissemination</a></li>
    
    </ul>
  </div>

  <div id="content">
    <h2>About me</h2>

<p>Between July 2021 and September 2022 I have joined <a href="https://www.uniklinikum-leipzig.de/einrichtungen/kinderklinik/Seiten/forschung-childhood-obesity.aspx">Antje KÃ¶rner&#39;s group</a>
at the <a href="https://www.uni-leipzig.de">University of Leipzig</a>, working on modelling risk for complex
disease predisposition and outcomes in childhood and identification of molecular
targets for complex disease from (multi)omics approaches.</p>

<p>Between March 2019 and June 2021 I have worked as an independent consultant,
specialised in biomedical
data analysis, biomarker discovery, and development of related software,
and as a visitor at the <a href="https://www.ed.ac.uk/igmm">Institute of Genetics and Molecular Biology</a>.
Between October 2009 and November 2018 I had been based in the
<a href="https://www.ed.ac.uk/usher/research-centres/population-health-sciences">Centre for Population Health Sciences</a>, part of the
<a href="https://www.ed.ac.uk/usher/">Usher Institute of Population Health Sciences and Informatics</a>,
as a Research Fellow and (from June 2016) as Senior Research Fellow in
<a href="https://pm2.phs.ed.ac.uk/">Paul McKeigue&#39;s group</a>.
Before that, I was a PhD student (from October 2003) and then a Postdoctoral
Research Assistant (from April 2007) within the <a href="https://www.maths.ed.ac.uk/ERGO/">Edinburgh Research Group in
Optimization</a> at the <a href="https://www.maths.ed.ac.uk/">School of Mathematics</a>.</p>

<h2>Research interests</h2>

<h3>Biomarker discovery and prediction of diabetes complications</h3>

<p><strong>November 2014 to June 2021:</strong> I have been working with <a href="https://www.ed.ac.uk/mrc-human-genetics-unit/research/colhoun-group">Helen Colhoun</a> on data
from the <a href="https://www.ed.ac.uk/scottish-diabetes-research-network-t1-bioresource">SDRNT1BIO cohort</a>, a large
cohort of patients with type 1 diabetes linked to electronic health data and
genotypes, and where proteins, metabolites, tryptic peptides, glycans are
measured in a subset of samples.
We investigated to the relationship of N-glycans with progression
of renal disease (<a href="https://doi.org/10.2337/dc17-1042">Diabetes Care 2018</a>, <a href="https://doi.org/10.1093/glycob/cwaa106">Glycobiology 2021</a>),
the relationship of persistent C-peptide secretion with the
genetic architecture of type 1 diabetes (<a href="https://doi.org/10.1186/s12916-019-1392-8">BMC Medicine 2019</a>),
the contemporary rates of progression of renal disease and their predictors
(<a href="https://doi.org/10.1007/s00125-019-05052-z">Diabetologia 2020a</a>), the validation of a cardiovascular risk
prediction model in type 1 diabetes (<a href="https://doi.org/10.1007/s00125-021-05478-4">Diabetologia 2021</a>),
and the risk factors for COVID-19 in people
with diabetes (<a href="https://doi.org/10.1016/S2213-8587(20)30405-8">Lancet Diabetes &amp; Endocrinology 2021</a>).
Our work on biomarkers of renal progression has produced the several results
(<a href="https://doi.org/10.1007/s00125-019-4915-0">Diabetologia 2019b</a>, <a href="https://doi.org/10.1007/s00125-019-05081-8">Diabetologia 2020b</a>,
<a href="https://doi.org/10.1111/pedi.13095">Pediatric Diabetes 2020</a>), and allowed us to investigate the effect
of storage conditions on the quality of metabolomic and proteomic studies
(<a href="https://doi.org/10.1038/s41598-022-08429-0">Scientific Reports 2022</a>).</p>

<p><strong>March 2012 to December 2015:</strong> I worked on the <a href="https://www.imi.europa.eu/projects-results/project-factsheets/summit">SUMMIT</a> project,
a European research consortium dedicated to diabetes complications, focusing
in particular on non-genetic biomarkers, data mining and in-silico
modelling. As complications, we principally looked at cardiovascular disease
(<a href="https://doi.org/10.1007/s00125-015-3535-6">Diabetologia 2015</a>, <a href="https://doi.org/10.1016/j.atherosclerosis.2018.05.014">Atherosclerosis 2018</a>) and progression of
diabetic kidney disease (<a href="https://doi.org/10.1038/ki.2015.199">Kidney International 2015</a>,
<a href="https://doi.org/10.1007/s00125-018-4741-9">Diabetologia 2019a</a>).
During the project we also tackled the problem of in-silico identification
of unknown metabolites (<a href="https://doi.org/10.1016/j.jchromb.2017.04.002">Journal of Chromatography B 2017</a>).</p>

<h3>Bioinformatics</h3>

<p><strong>February 2016 to now:</strong> I am one of the core developers of
<a href="https://pm2.phs.ed.ac.uk/genoscores/">GENOSCORES</a>, a platform built to provide a framework for calculating
genotypic predictors of binary and quantitative phenotypes from publicly
available summary results of genome-wide association studies of multiple
phenotypes, -omic measurements and gene expressions.
This was used to explore pleiotropy in the genetic determinants of male
pattern baldness (<a href="https://doi.org/10.1038/s41467-017-01490-8">Nature Communications 2017</a>), to discover two
new genetic loci strongly associated with response to anti-TNF in
rheumatoid arthrisis (<a href="https://doi.org/10.1136/annrheumdis-2018-214877">Annals of the Rheumatic Diseases 2019</a>), to
investigate the
relationship between persistent C peptide secretion in type 1 diabetes and the
genetic architecture of diabetes (<a href="https://doi.org/10.1186/s12916-019-1392-8">BMC Medicine 2019</a>), and provide a
novel approach to integrating genetics and neuroimaging in the exploration of
white matter microstructure (<a href="https://doi.org/10.1038/s41398-020-0724-y">Translational Psychiatry 2020</a>).</p>

<p><strong>December 2015 to November 2018:</strong> I have been involved in the <a href="http://www.matura.whri.qmul.ac.uk/">MATURA</a>
project, a consortium working on rheumathoid arthritis. The first results
concerned genome-wide association studies of response to methotrexate
(<a href="https://doi.org/10.1038/s41397-018-0025-5">The Pharmacogenomics Journal 2018a</a>) and tumour necrosis factor
inhibitor therapy (<a href="https://doi.org/10.1038/s41397-018-0040-6">The Pharmacogenomics Journal 2018b</a>), prediction of
response from genome-wide SNP data (<a href="https://doi.org/10.1002/gepi.22159">Genetic Epidemiology 2018</a>),
validity of a 2-component imaging-based disease activity score
(<a href="https://doi.org/10.1093/rheumatology/kez049">Rheumatology 2019</a>), and the discovery of potentially interesting new
associations (<a href="https://doi.org/10.1136/annrheumdis-2018-214877">Annals of the Rheumatic Diseases 2019</a>).</p>

<p><strong>January 2014 to December 2016:</strong> I worked the <a href="https://www.stratmed.co.uk/">Stratified Medicine Scotland
Innovation Centre</a> project PROMISERA, a study on early rheumatoid
arthritis. With the lead by <a href="http://www.homepages.ed.ac.uk/aspiliop/">Athina Spiliopoulou</a> we developed a novel
approach to imputation of ultralow coverage sequence data
(<a href="https://doi.org/10.1534/genetics.117.200063">Genetics 2017</a>), which is implemented in <a href="https://pm2.phs.ed.ac.uk/geneimp/">GeneImp</a>, which
relies on the existence of very large reference panel to avoid modelling
recombination explicitly. This leads to much faster imputations of this type
of data with minimal loss of quality.</p>

<p><strong>October 2009 to February 2012:</strong> I worked on the genetic epidemiology software
<a href="http://homepages.ed.ac.uk/pmckeigu/software/admixmap/">admixmap</a> with <a href="http://homepages.ed.ac.uk/pmckeigu/">Paul McKeigue</a>: I extended it to work correctly on
the X chromosome and we used this to better understand the genetic determinants
of sarcoidosis in Afro-American populations (<a href="https://doi.org/10.1038/gene.2010.56">Genes and Immunity 2011</a>).
Afterwards, I implemented a computationally efficient factorization that allows
to exploit pedigree data in hours rather than in weeks
(<a href="https://doi.org/10.1002/gepi.217101">Genetic Epidemiology 2013</a>).</p>

<h3>Large-scale optimization and structure exploitation</h3>

<p>My research on the theory and implementation of Interior Point Methods for
linear and quadratic programming concentrated particularly on the study of
search directions and warm-start approaches.</p>

<p><strong>April 2007 to September 2009:</strong> I was employed on an <a href="http://www.epsrc.ac.uk/">EPSRC</a>-funded
project with <a href="http://www.maths.ed.ac.uk/%7Eagr/">Andreas Grothey</a>, in which we continued the investigation
of warm-start strategies for interior point methods in the context of stochastic
programming.
This led to consider a multi-step approach, in which the number of intermediate
problems can be more than one (<a href="http://www.maths.ed.ac.uk/ERGO/pubs/ERGO-09-007.html">ERGO 09-007</a>), and a decomposition-like
strategy, in which we generate and warm-start the subproblems rooted at the
second-stage nodes (<a href="https://doi.org/10.1007/s10589-012-9530-7">COAP 2013</a>). I also designed the stochastic
programming extension for the structure-conveying modelling language <a href="http://www.maths.ed.ac.uk/ERGO/sml/">SML</a>
(<a href="https://doi.org/10.1007/s12532-009-0008-2">Mathematical Programming Computation 2009</a>).</p>

<p><strong>October 2003 to May 2007:</strong> During my PhD in optimization, under the
supervision of <a href="http://www.maths.ed.ac.uk/%7Egondzio/">Jacek Gondzio</a>, I have improved the implementation
of corrector directions in the <a href="http://www.maths.ed.ac.uk/%7Egondzio/software/hopdm.html">HOPDM</a> interior point solver
(<a href="https://doi.org/10.1007/s10589-007-9106-0">COAP 2008</a>), before moving to work with the structure-exploiting
parallel solver <a href="http://www.maths.ed.ac.uk/%7Egondzio/parallel/solver.html">OOPS</a>. I have implemented an <a href="https://github.com/mcol/libsmps">SMPS interface</a>
for OOPS, HOPDM and CPLEX (extended to LP_SOLVE and GLPK),
which allows to solve a stochastic programming problem by formulating the
corresponding deterministic equivalent problem. This implementation allows to
solve a problem instance with warm-start, by first solving a problem based on
a reduced scenario tree (<a href="https://doi.org/10.1007/s10107-009-0290-9">Mathematical Programming 2011</a>).</p>

  </div>

  <div class="meta">2003-2024 &copy; marco</div>
</div>

</body>
</html>
