<!DOCTYPE html>

<html>
<head>
  <title>Marco Colombo - Teaching</title>
  <meta charset="utf-8" />
  <meta name="author" content="Marco Colombo" />
  <meta name="description" content="Full steam research in Edinburgh" />
  <link rel="stylesheet" href="/css/style.css" />
  
  <link rel="icon" type="image/png" href="/pics/favicon.png" />
</head>

<body>

<div class="site">

  <div class="header">
    <h1 class="title"><a href="/">Marco Colombo</a></h1>
    <p class="tag">Experimental area &middot; proceed at your own risk!</p>
  </div>

  <div id="navigation">
    <ul>
    
    
      <li><a 
           href="/">Home</a></li>
    
      <li><a 
           href="/research/">Research</a></li>
    
      <li><a  class="active" 
           href="/teaching/">Teaching</a></li>
    
      <li><a 
           href="/software/">Software</a></li>
    
      <li><a 
           href="/engage/">Dissemination</a></li>
    
    </ul>
  </div>

  <div id="content">
    <h2>Biomedical Data Science</h2>

<p>Since 2016 I have been developing and teaching a 10 credits (20 hours) course
on analysis of biomedical data using the <a href="https://www.r-project.org/">R statistical software</a>
as part of the <a href="https://www.ed.ac.uk/maths/msc/or">MSc in Operational Research (with Data Science)</a>
and the <a href="https://www.ed.ac.uk/maths/msc/stats">MSc in Statistics</a>.</p>

<p>The course covers the following topics during 5 lectures (10 hours in total):</p>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lecture01.pdf">Introduction to biomedical data</a></h3>

<ul>
<li>Typical research questions: association, causation, discovery and prediction</li>
<li>Types of biomedical data: routine data (consented and unconsented), phenotypic
biomarkers, genetic data, derived data</li>
<li>Identifying problems in real-world data</li>
<li>Data cleaning, alignment, imputation and exploration</li>
<li>Mechanisms of missing data</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lecture02.pdf">Discovering associations</a></h3>

<ul>
<li>Covariance and correlation</li>
<li>Statistical inference and linear regression</li>
<li>Solving the least squares problem</li>
<li>Linear algebra considerations and collinearity</li>
<li>Hypothesis testing</li>
<li>Power considerations</li>
<li>Assessing the fit of the model</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lecture03.pdf">Logistic regression and predictive models</a></h3>

<ul>
<li>Case-control studies</li>
<li>Generalized linear models</li>
<li>Logistic regression</li>
<li>Odds ratio and interpretation of results</li>
<li>Likelihood and model comparison</li>
<li>Measures of discrimination and calibration performance</li>
<li>Predictive models and cross-validation</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lecture04.pdf">Biomarker discovery and high-dimensional datasets</a></h3>

<ul>
<li>High-throughput data (proteomics, metabolomics, lipidomics, glycomics)</li>
<li>Biomarkers and biomarker discovery</li>
<li>Dimensionality reduction: clustering and PCA</li>
<li>Multiple testing</li>
<li>Subset selection approaches</li>
<li>Penalised regression: LASSO, ridge regression, elastic nets</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lecture05.pdf">Prediction from genetic data</a></h3>

<ul>
<li>Causality, confounding and stratification</li>
<li>Introduction to genetic data</li>
<li>Genetic variation</li>
<li>Genome-wide association studies</li>
<li>GWAS meta-analysis</li>
<li>Approaches for genotypic prediction and genetic risk scores</li>
</ul>

<p>The course is accompanied by self-guided material to learn and practice how
to perform analyses using R (10 hours in total):</p>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lab01.pdf">Lab 1: Introduction to R</a></h3>

<ul>
<li>Interactive terminal and workspaces</li>
<li>Object types and data structures</li>
<li>Basic functions and operators</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lab02.pdf">Lab 2: Data preparation and linear regression</a></h3>

<ul>
<li>Merging and simple imputations</li>
<li>Statistical summaries and plots</li>
<li>Writing functions and loops</li>
<li>Fitting linear regression monels</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lab03.pdf">Lab 3: Logistic regression and predictive models</a></h3>

<ul>
<li>Using R packages</li>
<li>Fitting logistic regression models</li>
<li>Making predictions on withdrawn data</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lab04.pdf">Lab 4: High-dimensional datasets</a></h3>

<ul>
<li>Correlation plots and PCA</li>
<li>Subset selection in R</li>
<li>Regularisation approaches</li>
</ul>

<h3><a href="http://pm2.phs.ed.ac.uk/BDS/lab05.pdf">Lab 5: Prediction from genetic data</a></h3>

<ul>
<li>Performing genome-wide association studies</li>
<li>Computing genetic risk scores</li>
<li>Prediction from genetic scores</li>
<li>Performing a GWAS meta-analysis</li>
</ul>

  </div>

  <div class="meta">2016-2018 &copy; marco</div>
</div>

</body>
</html>
